CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 
  - #1530, Fix how the PostgREST version is shown in the help text when the `.git` directory is not available - @monacoremo
+ - #1603, Fix calling RPC with variadic argument by passing repeated params (#1470) - @wolfgangwalther
```

Comment:
My bad, I didn't notice the #1470 was at the end.

> I have been putting the PR with the change in front and related issues (could be more than one) at the end. Is that alright?

Until now, we've been putting the issue first, that way end users can understand the context of the fix/enhancement.
Sometimes when there's no issue and a direct PR is done, the PR number is put on the CHANGELOG.

> I have been putting the PR with the change in front and related issues (could be more than one) at the end. Is that alright?

Ideally it would be a PR per issue. Easier for users to read and easier for me to review ;)

Reply:
Ok, thanks for clearing that up. I changed that in all my open PRs and also created a small PR for those changelog entries that have already been committed with the wrong issue numbers.



> Ideally it would be a PR per issue. Easier for users to read and easier for me to review ;)

I know, the rpc-return stuff got quite big.... ;)
It's hard to slow myself down, when I'm on a roll and when stuff just depends on each other that leads to bigger PRs.